### PR TITLE
cephadm/box: fix remove image tar error

### PR DESCRIPTION
**Pull request by pereman2**
cleanups basically.

Fixes: https://tracker.ceph.com/issues/54045
Signed-off-by: Pere Diaz Bou <pdiazbou@redhat.com>

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [x] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by epuertat** [comment]
@pereman2 can you please document in Ceph docs how to add the 'shared folders' option?

File: src/cephadm/box/box.py [hunk 1]
```diff
@@ -39,9 +49,9 @@ def get_ceph_image():
     run_shell_command(f'docker build -t {CEPH_IMAGE} docker/ceph')
     if not os.path.exists('docker/ceph/image'):
         os.mkdir('docker/ceph/image')
-    image_tar = 'docker/ceph/image/quay.ceph.image.tar'
-    if not os.path.exists(image_tar):
-        os.remove(image_tar)
+
+    remove_ceph_image_tar()
+
     run_shell_command(f'docker save {CEPH_IMAGE} -o {image_tar}')
```
Comment (avanthakkar) [inline src/cephadm/box/box.py:55]:
```suggestion
    run_shell_command(f'docker save {CEPH_IMAGE} -o {CEPH_IMAGE_TAR}')
```

**Review by avanthakkar — CHANGES_REQUESTED** [review]

File: src/cephadm/box/box.py (hunk 1 above)
Comment (pereman2) [reply src/cephadm/box/box.py]: oops thanks!

**Review by nizamial09 — APPROVED** [review]
Tested locally! LGTM

**Comment by pereman2** [comment]
jenkins test make check

**Comment by pereman2** [comment]
jenkins test api

**Comment by pereman2** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Review by avanthakkar — APPROVED** [review]

**Pull request merged**